opencti-platform/opencti-graphql/tests/02-integration/08-httpServer/httpHealth-test.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, it } from 'vitest';
+import axios from 'axios';
+import { getBaseUrl } from '../../../src/config/conf';
+
+describe.skip('Should /health be available', () => {
+  // skipped because there is an issue with migration init.
+  // Please see testInitializeMigration in globalSetup
```

Comment:
since it's not working and not directly related to the taxii push test, should we remove it (with globalSetup changes) from this PR ?